modules/@apostrophecms/storybook/template/public/mocks.js

```diff
@@ -228,7 +314,7 @@
       url = apos.http.addQueryToUrl(url, options.qs);
     }
     if (apos.http.getResponses[url]) {
-      return apos.http.getResponses[url];
+      return JSON.parse(JSON.stringify(apos.http.getResponses[url]));
```

Comment:
There were bugs from the mock API because the response was not being cloned.

Reply:
I think this is worth a comment ("just like the response from a real API, the returned object should be safe to change").